src/lib/logger.js

```diff
@@ -19,28 +19,23 @@
 *
 */
 
-import { DefaultLogger } from './DefaultLogger';
-import { ILogger } from '../types';
+import bunyan from 'bunyan';
+import { ILoggerCreate, ILogger } from '../types';
+import path from 'path';
 
-export default class Logger {
-  static _logger: ILogger = DefaultLogger;
-
-  static error(...params: Array<any>) {
-    Logger._logger.error(...params);
-  }
-
-  static info(...params: Array<any>) {
-    Logger._logger.info(...params);
-  }
-
-  static initialize(logger: ILogger) {
-    Logger._logger = logger;
-  }
-
-  static log(...params: Array<any>) {
-    Logger._logger.log(...params);
+export default class Logger implements ILoggerCreate {
+  static createLogger(aName: string): ILogger {
```

Comment:
Don't use abbreviations ever -- if someone looks at this code later they would have no idea what `a` means. This should be `applicationName`